tutorials/save-query-output.md

```diff
@@ -5,4 +5,95 @@ nav_order: 2
 parent: Tutorials
 ---
 
-## Save the Output of a Query
+GlareDB provides a number of options to save the output of your queries,
+including GlareDB tables, temp tables, and views, and external files in a
+variety of formats.
+
+## Creating a GlareDB Table from a File
+
+For instance, say you have found a CSV file of user data on GitHub that you'd like
+to save to our GlareDB instance. You can run:
+
+```sql
+CREATE TABLE my_user_data AS
+SELECT * FROM read_csv(
+    'https://github.com/GlareDB/glaredb/raw/main/testdata/csv/userdata1.csv'
+);
```

Comment:
legitimate question: do you think it would be better to just do:

```sql
CREATE TABLE my_user_data AS
SELECT * FROM 'https://github.com/GlareDB/glaredb/raw/main/testdata/csv/userdata1.csv';
```

or do you think it benefits from having `read_csv` ?